provisioning/atat_provisioning.yaml

```diff
@@ -173,6 +181,18 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/PortfolioPatch'
+        '202':
+          description: Portfolio provisioning request has been accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProvisioningStatus'
+          headers:
+            Location:
+              description: Endpoint which ATAT should poll to receive status updates for this provisioning job
+              schema:
```

Comment:
Everything looks congruent with Open API 3.0, however as Kyle described in the Slack thread for the PR, the description should be updated to:
```
description: |
  Endpoint which ATAT should poll to receive status updates for this provisioning job.
  Data at this endpoint must be a valid ProvisioningStatus reponse.
```